playbooks/roles/airship-setup-deployer/tasks/main.yml

```diff
@@ -37,12 +37,17 @@
   tags:
     - install
 
+- name: Set the user name
+  set_fact:
+    user_name: "{{ lookup('env','USER') }}"
```

Comment:
(nit) Any particular reason for user_name as opposed to username which afaik is more commonly accepted?